### ISSUE TITLE
ALooper_pollAll is deprecated on Android NDK 27

### ISSUE THREAD

**Issue by Zombieschannel** [SFML/SFML#3181]
### Prerequisite Checklist

- [X] I searched for [existing issues](https://github.com/search?q=repo%3ASFML%2FSFML&type=issues) to prevent duplicates
- [X] I searched for [existing discussions on the forum](https://www.google.com/search?q=site%3Ahttps%3A%2F%2Fen.sfml-dev.org) to prevent duplicates
- [X] I am here to report an issue and not to just ask a question or look for help (use the [forum](https://en.sfml-dev.org/forums/index.php#c3) or [Discord](https://discord.gg/nr4X7Fh) instead)

### Describe your issue here

Basically everything [here](https://github.com/raysan5/raylib/issues/4166) applies to SFML (2 & 3) as well. In src/SFML/Window/Android/SensorImpl.cpp in the update function there is this line: `ALooper_pollAll(0, nullptr, nullptr, nullptr);` which seems to be deprecated. NDK 26 still compiles SFML fine, but NDK 27 throws out this error:
```
error: 'ALooper_pollAll' is unavailable: obsoleted in Android 1 - ALooper_pollAll may ignore wakes. Use ALooper_pollOnce instead. See The API documentation for more information
```

### Your Environment

SFML version: 2.6.1
Ninja
Android studio 2024.1.1
NDK: 27.0.11902837

### Steps to reproduce

Try to compile SFML for Android with NDK 27.

### Expected behavior

Compiles fine.

### Actual behavior

error: 'ALooper_pollAll' is unavailable: obsoleted in Android 1

**Comment by ChrisThrasher** [comment]
https://github.com/ChrisThrasher/SFML/actions/runs/10221306711/job/28283554085

I'm starting to see this sometimes fail CI jobs so we'll probably be forced to take action on this soon.

**Comment by ChrisThrasher** [comment]
@Zombieschannel Could you test this out for me? https://github.com/SFML/SFML/pull/3188

**Comment by ChrisThrasher** [comment]
Closed by #3188
